infrastructures/infrastructure-azure/build.gradle

```diff
@@ -11,9 +11,15 @@ targetCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
     compile project(':infrastructures:infrastructure-common')
-    compile "org.ow2.proactive:rm-server:${rmVersion}"
+    compile ("org.ow2.proactive:rm-server:${rmVersion}"){
+        exclude group: "com.fasterxml.jackson.core"
```

Comment:
you are using an old version of the code, please remove these exclusions as it would break current version